src/app/public/modules/repeater/repeater-item.component.scss

```diff
@@ -32,8 +32,13 @@
 
 .sky-repeater-item-left {
   display: flex;
-  margin-top: -2px;
   padding-top: $sky-padding;
+  align-items: center;
+  max-height: 45px;
```

Comment:
Why are we locking this down to `45px`?

Reply:
I've removed this. I was having some spacing issues and this was a style that stuck around from those. I've removed it and don't see any issues so I'm pretty sure that it isn't needed any longer.